addons/spreadsheet/static/src/pivot/plugins/pivot_ui_plugin.js

```diff
@@ -295,6 +295,9 @@ export class PivotUIPlugin extends spreadsheet.UIPlugin {
             if (!this.getters.isExistingPivot(pivotId) || !dataSource.isReady()) {
                 return undefined;
             }
+            if (!cell.content.replaceAll(" ", "").toUpperCase().startsWith("=ODOO.PIVOT.TABLE")) {
+                return undefined;
+            }
```

Comment:
I'd double check that we do'nt have similars bugs with other calls to `getFirstPivotFunction`  to be sure